### ISSUE TITLE
Syntax Error Sniff

### ISSUE THREAD

**Issue by nickdaugherty** [Automattic/VIP-Coding-Standards#289]
## What problem would the enhancement address for VIP?

The `WordPress-Extra` sniffs include syntax error detection, but since we don't extend that (just cherry pick out various WP sniffs via `ruleset.xml`, we are missing syntax errors.

test.php
```
<?php

thisisasyntaxerror
```

Running the ruleset produces no errors though:

```
$ phpcs --standard=WordPressVIPMinimum test.php
$
```

## Describe the solution you'd like

Syntax errors are caught by the `WordPressVIPMinimum` and `WordPress-VIP-Go` rulesets.

## What code should be reported as a violation?

Any PHP syntax error.

## What could should *not* be reported as a violation?

Valid PHP code.

## Additional context

We're missing this:

```
<rule ref="Generic.PHP.Syntax"/>
```

Right now we're picking certain rules out of the WordPress rulesets - would it make more sense to include the full ruleset and then exclude rules we don't want?

**Comment by davidsword** [comment]
This would be especially useful when using as a linter in an editor.
